<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1091</width>
      <height>431</height>
     </rect>
    </property>
    <widget class="QWidget" name="menu">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>621</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="text">
          <string>Welcome to NetQuiz! 
Click the button to join the game</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nickNameEdit">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>Your nick name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="connectButton">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="game">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>19</y>
        <width>1051</width>
        <height>411</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Player ranking</string>
         </property>
         <widget class="QListWidget" name="playerList">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>20</y>
            <width>521</width>
            <height>351</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="leaveGame">
         <property name="text">
          <string>Leave game</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="2">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="question">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>30</y>
            <width>421</width>
            <height>121</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>170</y>
            <width>340</width>
            <height>27</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="answer_0">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="answer_1">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="answer_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="answer_3">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="timerLabel">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>190</y>
            <width>341</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>0 seconds left</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QWidget" name="gridLayoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>250</y>
            <width>411</width>
            <height>151</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
            <widget class="QLineEdit" name="questionInput">
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
             <property name="placeholderText">
              <string>Question</string>
             </property>
            </widget>
           </item>
          <item row="1" column="0">
            <widget class="QLineEdit" name="answerA">
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
             <property name="placeholderText">
              <string>Answer A</string>
             </property>
            </widget>
           </item>
                      <item row="2" column="0">
            <widget class="QLineEdit" name="answerB">
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
             <property name="placeholderText">
              <string>Answer B</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLineEdit" name="answerC">
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
             <property name="placeholderText">
              <string>Answer C</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLineEdit" name="answerD">
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
             <property name="placeholderText">
              <string>Answer D</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QComboBox" name="correctAnswer">
          <property name="geometry">
           <rect>
            <x>430</x>
            <y>280</y>
            <width>86</width>
            <height>25</height>
           </rect>
          </property>
          <property name="currentText">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>430</x>
            <y>260</y>
            <width>91</width>
            <height>17</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Correct answer:</string>
          </property>
         </widget>
         <widget class="QPushButton" name="addQuestionButton">
          <property name="geometry">
           <rect>
            <x>430</x>
            <y>310</y>
            <width>91</width>
            <height>91</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Add question</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
